recipes/nrpys/meta.yaml

```diff
@@ -38,3 +37,7 @@ about:
   license: AGPL-3.0-or-later
   license_file: LICENSE.txt
   summary: "Python language bindings for nrps-rs substrate specificity predictor."
+
+extra:
+  additional-platforms:
+    - linux-aarch64       # openEuler-22.03-SP3 aarch64
```

Comment:
There is nothing specific to openEuler or any other Linux distribution. The comment should be removed.